lang/java/compiler/src/main/javacc/org/apache/avro/compiler/idl/idl.jj

```diff
@@ -81,7 +81,7 @@ import org.apache.avro.util.internal.Accessor;
 import com.fasterxml.jackson.databind.JsonNode;
 import com.fasterxml.jackson.databind.node.*;
 
-import org.apache.commons.lang3.StringEscapeUtils;
+import org.apache.commons.text.StringEscapeUtils;
```

Comment:
`org.apache.commons.lang3.StringEscapeUtils` is deprecated in favor of `org.apache.commons.text.StringEscapeUtils`.